README.md

```diff
@@ -59,9 +59,19 @@ wikibaseintegrator~=0.11.3
 # WikibaseIntegrator / WikidataIntegrator #
 
 WikibaseIntegrator (wbi) is a fork of [WikidataIntegrator](https://github.com/SuLab/WikidataIntegrator) (wdi) whose
-purpose is to focus on compatibility with Wikibase. There have been many improvements which have led to radical changes
-in the code. See the [release notes](https://github.com/LeMyst/WikibaseIntegrator/releases) to find out what has
-changed.
+purpose is to focus on an improved compatibility with Wikibase and adding missing functionalities.
+The main differences between these two libraries are :
+
+* A complete rewrite of the library with an object-oriented architecture allowing for easy interaction, data validation
+  and extended functionality
+* Add support for reading and writing Lexeme, MediaInfo and Property datatypes
```

Comment:
I think you are confusing data types with entity types. WDI according to its README does claim to support the Lexeme and Property data types. Also MediaInfo is not a built-in entity type, it is implemented by the [Extension:WikibaseMediaInfo](https://www.mediawiki.org/wiki/Extension:WikibaseMediaInfo) ... I think for everything supported by another extension you should mention which extension that is because users might not be familiar with the extra data/entity types.

Reply:
Yes, I mixed up datatype and entity.
WDI support Lexeme and Property as datatypes but not as entities, there's no WDPropertyEngine or WDLexemeEngine (like the WDItemEngine).